WordPress/WordPressStatsWidgets/Views/Localization/AppLocalizedString.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+
+extension Bundle {
+    static let app: Bundle = {
+        var url = Bundle.main.bundleURL
+        while url.pathExtension != "app" && url.lastPathComponent != "/" {
+            url.deleteLastPathComponent()
+        }
+        guard let appBundle = Bundle(url: url) else { fatalError("Unable to find the parent app bundle") }
+        return appBundle
+    }()
+}
```

Comment:
I pondered putting this in its own `Bundle+app.swift` dedicated file, to respect the "one class/extension per file" convention… but figure it was not much worth it.

Especially since, in the near future (in the soon-to-come PRs which will address other Widgets and App Extensions targets as I continue progressing on that project of mine), I plan to add that helper file to other targets (i.e. make this file be a member of not only the `WordPressStatsWidfets` but also all other app extensions needing app localization like the pre-iOS14 widgets or the `ShareExtension`) – as opposed to duplicate/recreate/re-type this code and helpers in each target.

Which means that having both the `Bundle.app` `extension` and the `AppLocalizedString` helper function in the same file and having only one file to add new targets as its target memberships will make things easier.

_PS: Obviously, when I get to that (in the future PRs) I'll start thinking on a better subdirectory to put that shared file in once I start sharing it with multiple targets_